ansible/roles/deploy/tasks/main.yml

```diff
@@ -1,8 +1,16 @@
 ---
+# tasks file for role/deploy
+- name: Stop container 
+  shell: "docker stop {{ container_name }}"
+  ignore_errors: true
+
+- name: Remove container
+  shell: "docker rm {{ container_name }}"
+  ignore_errors: true
 
 - name: Pull image from DockerHub
-  community.docker.docker_image:
-    name: ocaufcg/oca-frontend
-    source: pull
-    pull:
-      platform: amd64
+  shell: "docker pull {{ docker_image }}:latest"
+
+- name: Run container
+  shell: "docker run -d --name {{ container_name }} -p {{ container_port }}:{{ container_port }} {{ docker_image }}:latest"
+  ignore_errors: true
```

Comment:
I guess ignore_errors in this step is not interesting.
Maybe we can use the module docker_container
I has looking and it has a better error handling and also allows adding a restart policy.